src/index.ts

```diff
@@ -340,7 +340,10 @@ export async function getCodeFixesFromProject(project: Project, opt: Options, ho
 
 export function getDiagnostics(project: Project): (readonly Diagnostic[])[] {
   const diagnostics = project.program.getSourceFiles().map(function (file: SourceFile) {
-    return project.program.getSemanticDiagnostics(file);
+    return [
+      ...project.program.getDeclarationDiagnostics(file),
```

Comment:
I think that this needs to be conditional; I don't think that this will return `[]` if declaration mode isn't enabled (declaration/composite true?)

Reply:
Updated!